### PR TITLE
Set PMP to create invalid address

### PR DESCRIPTION
**Pull request by lz-bro**
Some hardware has full address coverage, so set PMP to create invalid address

Change-Id: I707281626521d321aadb6cee5aea457fc5c0424d

### CODE REVIEW

File: debug/testlib.py
Comment (aap-sc) [inline debug/testlib.py:1458]: do you really need this import here?

File: debug/testlib.py
Comment (aap-sc) [inline debug/testlib.py:1451]: I would double-check if we need to flush register cache after modification of pmp* stuff

File: debug/testlib.py
Comment (aap-sc) [inline debug/testlib.py:1454]: 1. can you add some indication in the log that an attempt resulted in a some kind of exception?
2. do you really need this `try` block since you've introduced this ` support_set_pmp_deny `  property for the target?

**Comment by lz-bro** [comment]
@aap-sc hello，I have made some modifications, could you please review it.

File: debug/gdbserver.py
Comment (aap-sc) [inline debug/gdbserver.py:302]: this looks like a typo t me, since you don't have a space.

**Comment by lz-bro** [comment]
@aap-sc Indeed, I fixed it.

**Review by aap-sc — APPROVED** [review]

**Comment by aap-sc** [comment]
@lz-bro lets LGTM from my side. Let's wait till monday before the merge (just to be sure that no one has objections)

**Pull request merged**
